collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -185,12 +185,6 @@
         RankingWidgetDialog.RankingListener,
         SaveFormIndexTask.SaveFormIndexListener, LocationClient.LocationClientListener, LocationListener {
 
-    // save with every swipe forward or back. Timings indicate this takes .25
-    // seconds.
-    // if it ever becomes an issue, this value can be changed to save every n'th
-    // screen.
-    private static final int SAVEPOINT_INTERVAL = 1;
```

Comment:
This change was introduced in 1778e150d86ebc722ef27d744f38019b6a7a5220 in 2012

It seems like this was added as a configurable setting to account for any future issues. Are we sure that this is safe to remove now?

Reply:
It was a final filed with value `1` used just for modulo and `x % 1` always returns 0 so it was redundant.